src/StarkEx.Client.SDK/Clients/Spot/SpotGatewayClient.cs

```diff
@@ -69,6 +69,16 @@ public async Task<int> GetTimeSpentByOldestUnacceptedTxInSystem(CancellationToke
         return await SendRequestAsync(requestModel, cancellationToken);
     }
 
+    public async Task<bool> IsAliveAsync(CancellationToken cancellationToken)
```

Comment:
Can you add an `/// <inheritdoc />` tag here? Now that we are at it the tag is also missing from the `AddTransactionAsync` method (DocFx does not comments to the docs otherwise).
![image](https://user-images.githubusercontent.com/98888090/219275785-414f18f9-c50a-42e3-aa37-191d68b33c4f.png)

Reply:
Meanwhile removed this method, was just using it for tests. I was thinking we could add the health check endpoints for all APIs in another PR